docs/user/mpi-io.md

```diff
@@ -119,6 +131,20 @@ export FI_UNIVERSE_SIZE=16383
 export FI_OFI_RXM_USE_SRX=1
 ```
 
+To use MPI-IO in Intel MPI with DAOS, the following two
+environment variables can be used to instruct the runtime
+to interpret all file paths in MPI-IO calls as DAOS paths:
+
+```bash
+export I_MPI_EXTRA_FILESYSTEM=on
+export I_MPI_EXTRA_FILESYSTEM_FORCE=daos
```

Comment:
looking at the link you posted bellow, it says those env variables are deprecated and they are replaced by:
I_MPI_FILESYSTEM
I_MPI_FILESYSTEM_FORCE

but i have not idea starting what version..

Reply:
thanks for catching this ... the impi 2021-11 documentation already mentions the new variables and states that if they are set, the old ones with "EXTRA" in their names are ignored.  i'll add this to the doc.